.gitignore

```diff
@@ -11,9 +11,18 @@ settings
 *.a
 *.pyd
 *.pyc
+path.txt
 /build
-/images
 /.sconsign.dblite
+/firmware/openos/bsp/boards/python/*.def

```

Comment:
@ftheoleyre we don't have the firmware folder in the repository. We have updated the structure and removed firmware folder. Could you remove it? thanks